[15.0][FIX] *: remove lxml, cryptography from python depends

EmilioSerna opened the pull request:
Such dependency is already included in Odoo requirements using a pinned
version [1][2]. Adding here could cause to upgrade the library to an
incompatible version.

[1] https://github.com/odoo/odoo/blob/710a2b2a7af68e8f2f249ef9fc3146f44d3266a5/requirements.txt#L18
[2] https://github.com/odoo/odoo/blob/710a2b2a7af68e8f2f249ef9fc3146f44d3266a5/requirements.txt#L3

luisg123v committed:
[FIX] html_text: remove `lxml` from python depends

Such dependency is already included in Odoo requirements using a pinned
version [1]. Adding here could cause to upgrade the library to an
incompatible version.

[1] https://github.com/odoo/odoo/blob/54e58b3e47ee/requirements.txt#L23

luisg123v committed:
[FIX] html_image_url_extractor: remove `lxml` from python depends

Such dependency is already included in Odoo requirements using a pinned
version [1]. Adding here could cause to upgrade the library to an
incompatible version.

[1] https://github.com/odoo/odoo/blob/54e58b3e47ee/requirements.txt#L23

EmilioSerna commented:
Hi @luisg123v 

Could you please review?

cc @hugho-ad

luisg123v reviewed (changes requested):
Remove also `cryptography`. It was not removed from v14 because it was not in Odoo's main requirements (see https://github.com/OCA/reporting-engine/pull/757#issuecomment-1516833160). But, starting from v15, it is.

EmilioSerna committed:
[FIX] letsencrypt: remove `cryptography` from python depends

Such dependency is already included in Odoo requirements using a pinned
version [1]. Adding here could cause to upgrade the library to an
incompatible version.

[1] https://github.com/odoo/odoo/blob/710a2b2a7af68e8f2f249ef9fc3146f44d3266a5/requirements.txt#L3

EmilioSerna commented:
> Remove also `cryptography`. It was not removed from v14 because it was not in Odoo's main requirements (see [OCA/reporting-engine#757 (comment)](https://github.com/OCA/reporting-engine/pull/757#issuecomment-1516833160)). But, starting from v15, it is.

@luisg123v 
Changes applied

luisg123v reviewed (changes requested):
Combine the two last commits (letsencrypt and the one that removes cryptography from requirements) because they both belong to the same change. Take into account the `requirements.txt` file is autogenerated from module's manifest

EmilioSerna commented:
@luisg123v 

Done

luisg123v reviewed (approved):
LGTM :+1: 

@moylop260 could you review, please?

moylop260 commented:
/ocabot merge patch

yaazkal commented:
I know this is old now, but I wonder why you are keeping `cryptography==2.6.1` anyway in the `requirements.txt` file and in the `auto_backup` module.
